app/models/courseNameAndNumber.py

```diff
@@ -0,0 +1,17 @@
+from app.models import * 
+
+def nameNumCombo(name, abbreviation):
```

Comment:
This function is not a model. It should either go in a `logic/` file, or in the `Course` model